debian_packages.lst

```diff
@@ -3,10 +3,14 @@ git
 libffi-dev
 libpq-dev
 make
+python3.6
 python3.6-dev
 python3-pip
+python3.6-venv
 redis-server
 subversion
 xvfb
 postgresql-client
 postgresql
+wkhtmltopdf
```

Comment:
If this is present as a package, why does it need to be downloaded manually?

Reply:
The package for this is highly incomplete. See the warning @ https://github.com/JazzCore/python-pdfkit#installation. I recall trying to test it and it didn't work as I hoped. I can remove it now to not confuse anyone.

Reply:
@antoviaque Actually, this is required: it installs a huge list of shared dependencies which without the binary fails to execute. I noticed it when moving to the new OpenStack account/region without having this -- Alicia reported a 500 error, it was due to this missing these dependencies.

Reply:
@UmanShahzad Maybe add a comment about this next to it then, so we can remember about it reading this?

Reply:
@antoviaque Yes, good idea.